files/gitignore

```diff
@@ -7,12 +7,12 @@
 .sass-cache
 .eslintcache
 coverage/
-libpeerconnection.log
 npm-debug.log*
 yarn-error.log
 
 # ember-try
 /.node_modules.ember-try/
 /package.json.ember-try
+/package-lock.json.ember-try
 /yarn.lock.ember-try
-/pnpm-lock.yaml.ember-try
+/pnpm-lock.ember-try.yaml
```

Comment:
Use the correct extension:
https://github.com/ember-cli/ember-try/blob/master/lib/dependency-manager-adapters/pnpm.js#L15C31-L15C55